Add back overview (#1364)
astronomer/docs: astro/features.md

---
sidebar_label: 'Features'
title: 'Astro features'
id: environment-variables
description: Set environment variables on Astro to specify Airflow configurations and custom logic.
---

Astro is a managed software service that offers a next-generation experience for modern data teams running Apache Airflow, the open source industry standard for data orchestration.

Astro boasts a hybrid deployment model founded on a control plane hosted by Astronomer and a data plane that is hosted in your cloud environment. Both are fully managed by Astronomer. This model offers the self-service convenience of a fully managed service while respecting the need to keep data private, secure, and within corporate boundaries. This solution optimizes security and reduces operational overhead.

> Beyond architecture, Astro offers a suite of first-class features that make it easy to author, run, and monitor data pipelines.

Reviewer: ```suggestion
Beyond architecture, Astro offers a suite of first-class features that make it easy to author, run, and monitor data pipelines.
```
![image](https://user-images.githubusercontent.com/104205257/196740281-a887d194-2eac-4fa0-868f-1eb73666b00e.png)

Reviewer 2: Let's stick with "easy" -- I wouldn't use that word in documentation ("follow step 1, it's easy!") but I think in a feature overview of our product it's appropriate to underscore the value we bring to market. And that is.. "making all this stuff easy."

## Feature list

> The key features of Astro include:

Reviewer: ```suggestion
The key features of Astro include:
```
Remove this introductory statement. It's not necessary.

> - Worker auto-scaling, powered by the Airflow Celery executor + KEDA. See [Worker queues](configure-worker-queues.md).

Reviewer: ```suggestion
- Worker auto-scaling, powered by the Airflow Celery executor and KEDA. See [Worker queues](configure-worker-queues.md).
```

- The ability to run multiple clusters in your organization's network on AWS, GCP, or Azure.
- Astro Runtime, a collection of Docker images that provides a differentiated data orchestration experience. See [Runtime image architecture](runtime-image-architecture.md).
- Timely support for the latest major, minor, and patch versions of the Apache Airflow open source project.
- Support for role-based access control (RBAC) and single sign-on (SSO) for secure user management and authentication. See [Configure an identity provider (IdP)](configure-idp.md) and [User permissions](user-permissions.md).

> - An observability experience that gives you insight into the health and resource consumption of your tasks and pipelines in real time. See [Deployment metrics](deployment-metrics.md) and [Data lineage](data-lineage.md).

Reviewer 2: ```suggestion
- An observability experience that gives you data lineage across your ecosystem as well as real-time insight into the health and resource consumption of your data pipelines on Astro. See  [Data lineage](data-lineage.md] and [Deployment metrics](deployment-metrics.md).
```

> - An observability experience that gives you insight into the health and resource consumption of your tasks and pipelines in real time. See [Deployment metrics](deployment-metrics.md) and [Data lineage](data-lineage.md).

Reviewer: ```suggestion
- Insight into the health and resource consumption of your tasks and pipelines in real time. See [Deployment metrics](deployment-metrics.md) and [Data lineage](data-lineage.md).
```
